common.mk

```diff
@@ -73,15 +73,15 @@ linux-common: linux-defconfig
 linux-defconfig-clean-common:
 	@rm -f $(LINUX_PATH)/.config
 
-LINUX_CLEAN_COMMON_FLAGS ?= $(LINUX_COMMON_FLAGS)
+#LINUX_CLEAN_COMMON_FLAGS ?= UNCOMMENT_AND_DEFINE_AS_NECESSARY
```

Comment:
@vchong , maybe the `CROSS_COMPILE` isn't needed, it may very well be the case. Usually there is no need to include that variable when cleaning. One can just test it:
```
build && make clean && git clean -xfdn > only_clean.txt 
build && make CROSS_COMPILE=... clean && git clean -xfdn > cc_clean.txt 
diff -Naur only_clean.txt cc_clean.txt
```
If there's no diff, then